diagnostics/flow_dep_diag/flow_dep_diag.yml

```diff
@@ -0,0 +1,103 @@
+name: MDTF_flow_dep_diag
```

Comment:
This YAML is overly prescriptive.  At a minimum, remove the build codes.  Consider just listing the main top-level packages rather than every dependency.

Reply:
@jkrasting  Is there a way to automatically generate the yaml file with only top-level packages. I know at least for removing the build codes I can use the --no-builds flag.

Reply:
@drewmresnick you can try using `conda env export --from-history flow_dep_diag.yml` per [this SO reply](https://stackoverflow.com/a/58487556). However, the post states that this will not include package versions. Otherwise, you'll have to modify the file manually, as conda does not seem to have this exact capability.